versioned/persist/serialize/src/main/proto/persist.proto

```diff
@@ -77,10 +77,34 @@ message GlobalStateLogEntry {
   repeated ContentIdWithBytes puts = 4;
 }
 
+message RefLogEntry {
```

Comment:
Rename this one to avoid fully-qualified class names - clashes with `org.projectnessie.versioned.RefLogEntry` and  `org.projectnessie.model.RefLogResponse.RefLogEntry`

Reply:
changed now.
proto is having RefLogEntry
RefLogResponse is having RefLogResponseEntry
spi is having RefLog